cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/harbor_api_v2_repo_lister.go

```diff
@@ -0,0 +1,298 @@
+// Copyright 2021-2022 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"unicode"
+
+	httpclient "github.com/vmware-tanzu/kubeapps/pkg/http-client"
+	"helm.sh/helm/v3/pkg/registry"
+	log "k8s.io/klog/v2"
+	"oras.land/oras-go/v2/errdef"
+	orasregistryv2 "oras.land/oras-go/v2/registry"
+	orasregistryauthv2 "oras.land/oras-go/v2/registry/remote/auth"
+)
+
+// This flavor of OCI repsitory ister works with respect to to VMware Harbor
+// container registry. The Swagger for the API can be found on an running server
```

Comment:
Another nitpick: the `Swagger` name is discouraged in favor of "OpenAPI"

```suggestion
// container registry. The OpenAPI description of the API can be found on a running server
```

Reply:
link to source please? 
Harbor certainly did not get the message
<img width="1440" alt="Screen Shot 2022-09-06 at 8 18 02 AM" src="https://user-images.githubusercontent.com/74032303/188673340-69cbab5e-dc64-449c-bd69-17e45d4ff143.png">

Reply:
Sorry, here it is: https://www.openapis.org/faq/style-guide.
Besides, my former company was a member of the OpenAPI Initiative and I was somehow also involved in.

But... the reality is that there are plenty of people still using the "Swagger" word. In fact, it can be a synonym, but only if the spec version is `2.0`. 

In fact, the according to the style guide, the proper term would be `OpenAPI Specification document` or just `OAS document`

Reply:
thanks. I will re-word comment